src/main/k6StudioState.ts

```diff
@@ -1,16 +1,17 @@
-import { ChildProcess } from 'child_process'
 import { FSWatcher } from 'chokidar'
 import { BrowserWindow } from 'electron'
 import eventEmitter from 'events'
 
+import { RecordingSession } from '@/handlers/browser/recorders/types'
+
 import { ProxyStatus } from '../types'
 import { AppSettings } from '../types/settings'
 
 import { type ProxyProcess } from './proxy'
 import { defaultSettings } from './settings'
 
 export type k6StudioState = {
-  currentBrowserProcess: ChildProcess | null
+  currentRecordingSession: RecordingSession | null
```

Comment:
Use an abstraction instead of working with the `ChildProcess` directly. This solves the problem of disposing e.g. the browser server.